src/app/model/bodies/sensorTheme.ts

```diff
@@ -1,9 +1,19 @@
 export enum SensorTheme {
-    Wheather = 'Wheather',
+    Weather = 'Weather',
     NatureAndEnvironment = 'NatureAndEnvironment',
     Waste = 'Waste',
     Safety = 'Safety',
     Mobility = 'Mobility',
     SoilandUnderground = 'SoilandUnderground',
     Other = 'Other',
 }
+
+export const SensorThemeTranslation = {
```

Comment:
Met de extra Translation consts vraagt het extra werk om die in sync te houden met de enum waar de vertalingen bijgevoegd worden. Ik snap dat dit nodig is om altijd de Engels naam op te slaan, maar de gelocaliseerde versie te tonen. 

Heb je echter ook gekeken of we beiden kunnen combineren om duplication te beperken?

Reply:
Yes, wij gebruiken het SensorTheme enum als type binnen een aantal objecten, die zal bewaard moeten blijven. Helaas kan niet rechtstreeks de vertaling gebruikt worden als value van deze enum, omdat deze berekend moet worden 'Computed values are not permitted in an enum with string valued members.'. Terwijl het mij wel mooi lijkt om een enum type te gebruiken voor categorie / type / thema. Daarom een extra lookup die gebruikt wordt voor de vertaling. Of weet jij een manier om dit te kunnen combineren?

Reply:
Nee, schoot mij zo ook nog niks te binnen, dit was ook hoe ver ik was. Ben het ermee eens dat de enum ook echt waarde toevoegd, zonde om dat er nu uit te halen. Ik had gehoopt dat er misschien nog een Typescript type zou zijn die zoiets kon combineren. Anyway, prima om te laten staan zo, staat nu in ieder geval bij elkaar in de code, dat houdt voldoende bij elkaar 😊